src/ostorlab/agent/mixins/agent_report_vulnerability_mixin.py

```diff
@@ -42,14 +90,13 @@ class VulnerabilityLocation:
     metadata: List[VulnerabilityLocationMetadata]
     asset: Union[os_asset.Asset, None] = None
 
-    def to_dict(self) -> Dict[str, Union[Any, List[Dict[str, str]]]]:
+    def to_dict(
```

Comment:
can we add the order to the dict here since we will need that too?

Reply:
The sorting logic has nothing to do with the message itself, it's related to the computation of DNA, which is an agent-specific thing.